docs/modules/reference/pages/billing/signup/private-cloud.adoc

```diff
@@ -0,0 +1,172 @@
+
+=== 2. Azure Configuration
+
+. Access Azure Marketplace:
+// TODO: Link will be different since cloud-cmr uses standard Azure contract, which we will not use in the final version
+.. Search for https://portal.azure.com/#create/payara.cloud-cmr[Payara Cloud Customer Managed Region]
+.. Select the available plan
++
+// TODO: This needs to be updated with final public plan screenshot
+image::billing/signup/cmr-planselect.png[Plan selection]
+
+. Configure basic settings:
++
+image::billing/signup/cmr-azureparams-1.png[Basic configuration]
++
+Subscription:: Azure subscription for deployment
+Resource Group:: Container for managed application
+Region:: Azure deployment location
+Connection String:: Value copied from Payara Cloud
+Domain:: Your DNS subdomain
+Let's Encrypt Email:: SSL certificate notifications
+Application Name:: Managed Application identifier
+Managed Resource Group:: Resource container name
+
+. Set infrastructure options (Optional):
++
+image::billing/signup/cmr-azureparams-2.png[Infrastructure settings]
++
+AKS Cluster Name:: Azure Kubernetes Service cluster identifier
+Kubernetes Version:: Platform version selection
+Node Size:: VM type (4+ vCPUs recommended)
+Cluster Size Range::
+* Minimum: Base node count
+* Maximum: Scale limit
+Log Analytics Workspace:: Monitoring workspace name
+
+. Review and create:
+.. Verify all settings
++
+image::billing/signup/cmr-azureparams-3.png[Review settings]
+
+.. Click btn:[Create]
+
+. Monitor deployment:
+.. Watch progress indicators
++
+image::billing/signup/cmr-azuredeployment-1.png[Deployment progress]
+
+.. Check outputs after completion
++
+image::billing/signup/cmr-azuredeployment-2.png[Deployment outputs]
++
+TIP: Note the `dns_nameservers` values for the next step
+
+=== 3. DNS Setup
+
+. Configure domain settings:
+.. Create NS records for your subdomain
+.. Add all `dns_nameservers` values
++
+image::billing/signup/cmr-azuredeployment-3.png[DNS configuration]
++
+.Example DNS Configuration
+[source]
+----
+Subdomain: [your-region-name]
+Record type: NS
+TTL: 3600
+Values: [dns_nameservers values]
+----
+
+=== 4. Final Verification
+
+. Check Payara Cloud dashboard:
+.. Monitor region status
+.. Wait for "Active" indication
++
+image::billing/signup/cmr-verification-1.png[Region verification]
+
+NOTE: This subscription will only show your private regions. Public cloud regions are not accessible.
+
+== Troubleshooting Guide
+
+.Common Issues
+* Region activation delay
+** Normal time: 10-15 minutes
+** If exceeding 30 minutes, verify:
+*** DNS configuration accuracy
+*** Azure deployment status
+*** Connection string correctness
+
+== Next Steps
+
+. Create namespaces in your new region
+. Deploy your first application
+. Monitor Azure portal metrics
+
+TIP: Keep your Azure infrastructure costs optimized by monitoring resource usage regularly.
+
+== Support
+
+For technical assistance:
+
+* Check Payara Cloud documentation
```

Comment:
Suggestion: A link to `How-To-Guides` or documentation home page would be nice improvement